python/paddle/distributed/fleet/meta_parallel/pipeline_parallel.py

```diff
@@ -53,12 +104,15 @@ def __init__(self, layers, hcg, strategy):
         self.stage_id = self._hcg.get_stage_id()
         self.pp_group = self._hcg.get_pipe_parallel_group()
 
+        global _REAL_PP_WORLD_SIZE
```

Comment:
Why we must need a global variable?

Reply:
Has move all these checks into PipelineParallel